File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/CommandMetadata.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class CommandMetadata{

	@SerializedName("webCommandMetadata")
	private WebCommandMetadata webCommandMetadata;

	public void setWebCommandMetadata(WebCommandMetadata webCommandMetadata){
		this.webCommandMetadata = webCommandMetadata;
	}

	public WebCommandMetadata getWebCommandMetadata(){
		return webCommandMetadata;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/CommandMetadata.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class CommandMetadata : Parcelable, Serializable {
    @SerializedName("webCommandMetadata")
    var webCommandMetadata: WebCommandMetadata? = null
    override fun toString(): String {
        return "CommandMetadata{" +
                "webCommandMetadata = '" + webCommandMetadata + '\'' +
                "}"
    }
}